var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -104,6 +100,8 @@ class CbtfKrell(Package):
 
```

Comment:
We build multiple MPI implementation collectors into our tool.  So, one build supports multiple types of MPI.   It is currently working the way we want it to.   We only build a portion of our tool using the MPI implementation libraries and include files.   This needs to stay the way it is for our tool to function the way we intend it to.